src/main/java/com/epam/university/java/core/task026/Task026Impl.java

```diff
@@ -0,0 +1,109 @@
+package com.epam.university.java.core.task026;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Created by ilya on 30.09.17.
+ */
+public class Task026Impl implements Task026 {
+
+    private final static Map<Integer, String> INT_TO_STRING = new HashMap();
+    private final static int ALFABET_LENGTH = 26;
+    private final static Map<String, Integer> STRING_TO_INT = new HashMap();
+
+    static {
+        INT_TO_STRING.put(0, "a");
```

Comment:
Hmmm, each character have its own index, it's better to use it instead of enumeration this way.